etc/worker/Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG GO_VERSION
 FROM golang:${GO_VERSION} AS worker_build
 WORKDIR /app
-COPY init.go .
+COPY etc/worker/init.go .
```

Comment:
same comment as above: Let's make the thing that builds this image cd into the right working directory for the build context to relative be `./`, if possible?